Telerik.JustMock/Expectations/Abstraction/INonPublicExpectation.cs

```diff
@@ -44,14 +44,25 @@ public interface INonPublicExpectation
 		/// <returns>Reference to setup actions calls</returns>
 		ActionExpectation Arrange(object target, string memberName, params object[] args);
 
-		/// <summary>
-		/// Setups a non-public method for mocking.
-		/// </summary>
-		/// <param name="target">Target instance</param>
-		/// <param name="method">Method to setup taken from reflection.</param>
-		/// <param name="args">Method arguments</param>
-		/// <returns>Reference to setup actions calls</returns>
-		ActionExpectation Arrange(object target, MethodInfo method, params object[] args);
+
+        /// <summary>
+        /// Arranges a method for mocking.
```

Comment:
I agree with Ivo. I saw that this is the description used for the rest of the overloads that were already there. Nevertheless, it would be very helpful if we could extend the provided information to be more meaningful.

Reply:
I'll update the documentation once we have stable build.